app/forms/import_form.rb

```diff
@@ -0,0 +1,24 @@
+class ImportForm
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+  include ActiveModel::Validations
+
+  ATTRIBUTES = [:url, :preview]
+  attr_accessor *ATTRIBUTES
+
+  validates_presence_of :url
+
+  def persisted?
+    false
+  end
+
+  def preview?
+    preview == "1"
+  end
+
+  def initialize(attributes = {})
```

Comment:
Okay, looking into this.  It seems to work and lets me drop a lot of code, I just want to understand what all that gives me.  (My FBO knowledge is clearly from Rails 3 or so)